R/helper-quarto.R

```diff
@@ -16,44 +17,49 @@ quarto_multilingual_book <- function(parent_dir,
                                      main_language = "en",
                                      further_languages = c("es", "fr"),
                                      register_languages = TRUE,
-                                     site_url = "https://example.com") {
+                                     site_url = "https://example.com",
+                                     placement = "sidebar") {
```

Comment:
```suggestion
                                     placement = c("sidebar", "navbar")) {
```

Realizing that then inside the body of the function we could use https://rlang.r-lib.org/reference/arg_match.html

Reply:
I thought about that when writing it! Actually about using `arg.match()` but it's the same.